### PR TITLE
Adding to the set climate timer service as part of code pr

### PR DESCRIPTION
**Pull request by north3221**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/65886
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by north3221** [comment]
@michaelarnauts sorry can you approve the doc update too please?

File: source/_integrations/tado.markdown [hunk 1]
```diff
@@ -123,8 +123,10 @@ You can use the service `tado.set_climate_timer` to set your Tado climate device
 | Service data attribute | Optional | Description                                                            |
 | ---------------------- | -------- | ---------------------------------------------------------------------- |
 | `entity_id`            | yes      | String, Name of entity e.g., `climate.heating`                         |
-| `time_period`          | no       | Time Period, Period of time the boost should last for e.g., `01:30:00` |
 | `temperature`          | no       | String, The required target temperature e.g., `20.5`                   |
+| `time_period`          | yes      | Time Period, Period of time the boost should last for e.g., `01:30:00` |
+| `temperature`          | yes      | Override your defaults setting. NB dont set this and the time period   |
```
Comment (frenck) [inline source/_integrations/tado.markdown]: Temperature is now listed twice.

Comment (north3221) [reply source/_integrations/tado.markdown]: What an idiot! Sorry good spot.. Updated

**Comment by frenck** [comment]
Overlay is new? That isn't in the current stable release, however, this PR targets the `current` branch and isn't linked to a core PR?

Is there a PR against core adding this functionality?

**Comment by north3221** [comment]
> Overlay is new? That isn't in the current stable release, however, this PR targets the `current` branch and isn't linked to a core PR?
> 
> Is there a PR against core adding this functionality?

Sorry not done very well here, how do I update it to be next branch? Yes the PR for the functionality is here:
https://github.com/home-assistant/core/pull/65886

**Comment by frenck** [comment]
Please add the link/reference to the PR description (it was asked for in the PR template).
You also need to change the target branch of this PR from `current` to `next`. After that, you might need to git rebase this PR onto the  `next` branch to clean itup.

**Comment by north3221** [comment]
> Please add the link/reference to the PR description (it was asked for in the PR template). You also need to change the target branch of this PR from `current` to `next`. After that, you might need to git rebase this PR onto the `next` branch to clean itup.

Sorry I made of mess of trying to do that online so opened new, clean, pr for it against next: https://github.com/home-assistant/home-assistant.io/pull/21660

Closing this one

**Pull request closed without merging**